source/board/vbluno51.c

```diff
@@ -32,9 +32,9 @@ const char *board_id = "C006";
 __attribute__((aligned(4)))
 const vfs_filename_t daplink_url_name =       "VBLUNO51HTM";
 __attribute__((aligned(4)))
-const vfs_filename_t daplink_drive_name =     "DAPLINK    ";
+const vfs_filename_t daplink_drive_name =     "VBLUNO51   ";
```

Comment:
This name may not be properly detected with mbed-ls. CC @theotherjimmy

Reply:
Yep, probably would not be recognized by mbed-ls

Reply:
Do I have to use mbed.htm instead of vbluno51.htm or use DAPLINK for drive name?

Reply:
DAPLINK or MBED should work

Reply:
OK. I will rebase this commit